public_html/assets/css/nf-core.css

```diff
@@ -921,7 +921,7 @@ Based on https://codepen.io/wintr/pen/beBJBb */
 }
 
 .fa_icon_picker.popover {
-  width: 17rem;
+  width: 15.3rem;
```

Comment:
How come this change was needed? For me the size already fits the contents?

![image](https://user-images.githubusercontent.com/465550/77297921-4e4c6d00-6cea-11ea-8097-04664ab61b22.png)

Reply:
My Chrome:
<img width="302" alt="Screenshot 2020-03-23 12 09 00" src="https://user-images.githubusercontent.com/6169021/77310953-801bfe80-6cff-11ea-8ea6-fd8000c3c4a1.png">
Could it maybe be because I don't hide my scrollbars?

Reply:
Ah, yes - that'll be it 👍 

Maybe we could force the scrollbar to always show? Would be a good idea anyway to show that there are lots of icons to choose from and not just the ones initially visible..

https://codepen.io/stevenlewis/pen/hubpL

Reply:
I tried this and several other versions and at least for me it doesn't work after changing the general settings to show scroll bars only while scrolling. I guess it is best to keep you width then, which works for standard settings.

Reply:
ok!